clients/cobol-lsp-vscode-extension/README.md

```diff
@@ -2,16 +2,16 @@
 
 [![Build Status](https://ci.eclipse.org/che4z/buildStatus/icon?job=LSP+for+COBOL%2Fdevelopment)](https://ci.eclipse.org/che4z/job/LSP%20for%20COBOL/job/development/)
 [![GitHub issues](https://img.shields.io/github/issues-raw/eclipse/che-che4z-lsp-for-cobol)](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues)
-[![slack](https://img.shields.io/badge/chat-on%20Slack-blue)](https://join.slack.com/t/che4z/shared_invite/zt-vlw321sg-2HuUaU0zI7fGIo2q7sEKZg)
+[![slack](https://img.shields.io/badge/chat-on%20Slack-blue)](https://communityinviter.com/apps/che4z/code4z)
```

Comment:
I have updated the read-me from the latest development branch.
@zimlu02 @zeibura Is it ok?

Reply:
Please leave that to me in future, even if it wasnt okay I wouldnt be able to fix it on your fork

Reply:
Just merge this and I will check the readme and changelog before release